src/lib/functions/server.js

```diff
@@ -105,6 +106,16 @@ const createHandler = function ({ functionsRegistry }) {
       const { error } = await func.invoke(event, clientContext)
 
       handleBackgroundFunctionResult(functionName, error)
+    } else if (await func.isScheduled()) {
+      if (event.headers['user-agent'] !== 'netlify-cli') {
+        return response.status(400).send('Scheduled functions can only be invoked using `netlify functions:invoke`.')
```

Comment:
[pebble] Since we know the name of the function, how do you feel about giving people the full command, i.e. `netlify functions:invoke <name of function>`?

Reply:
https://github.com/netlify/cli/pull/3689/commits/c61a2145c1a2f7e5575a9feb61d136a5ad70cba6#diff-de106b743be68ae2e9146d09931f894e258783a68db916a6d0e39e3908b28c84R114